docs/content/Infrastructure.md

```diff
@@ -136,6 +136,11 @@ Shared modules between android-test and gradle.
 
 ## Publishing
 
+### Versioning
+
+[projectVersion](https://github.com/avito-tech/avito-android/blob/develop/gradle.properties#L13) 
+is version for future release. `projectVersion` in commit means in what release version commit will appear.
```

Comment:
`projectVersion in commit means in what release version commit will appear` - looks like unnecessary explanation.